packages/mobile/src/components/comments/CommentBlock.tsx

```diff
@@ -34,7 +34,7 @@ export const CommentBlock = (props: CommentBlockProps) => {
     isPinned,
     message,
     reactCount = 0,
-    trackTimestampS,
+    trackTimestampMS,
```

Comment:
merge quirk, this should be the before
```suggestion
    trackTimestampS,
```